projects/exercises/01/drive_in_line.cpp

```diff
@@ -0,0 +1,12 @@
+#include <STSL/RJRobot.h>
+
+int main()
+{
+    RJRobot robot(); //Initialize a RJ Robot
```

Comment:
```suggestion
    RJRobot robot; //Initialize a RJ Robot
```

Last change I think we need before merging, but this line actually invokes the [most vexing parse](https://en.wikipedia.org/wiki/Most_vexing_parse). The suggested change should fix that.